### PR TITLE
update php req to at least 7.3

### PR DESCRIPTION
**Pull request by samsonasik**
Based on @michalbundyra comment at https://github.com/laminas/laminas-mvc-skeleton/pull/17#issuecomment-612363466

|    Q          |   A
|-------------- | ------
| Documentation | yes

### CODE REVIEW

File: docs/book/getting-started/overview.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This tutorial is intended to give an introduction to using Laminas by creating a
 
 ## Some assumptions
```
Comment (froschdesign) [inline docs/book/getting-started/overview.md:5]: The entire section "Some assumptions" should be removed and the system requirements should be added to the page "[Getting started: A skeleton application](https://docs.laminas.dev/tutorials/getting-started/skeleton-application/)".

**Comment by samsonasik** [comment]
I removed "Some assumption" part and add requirement at least php 7.3 to getting started.

**Comment by samsonasik** [comment]
do you think apache config need to be moved to getting started skeleton ?

**Comment by froschdesign** [comment]
@samsonasik 
> do you think apache config need to be moved to getting started skeleton ?

Is already there. See under "[Using the Apache Web Server](https://docs.laminas.dev/tutorials/getting-started/skeleton-application/#using-the-apache-web-server)".

**Comment by michalbundyra** [comment]
Thanks, @samsonasik!

**Comment by froschdesign** [comment]
@samsonasik 
I've overlooked that not the entire section "Some assumption" was removed. The blockquote "Getting started faster" must also be deleted.

**Pull request merged**